docs/content/docs/connectors/datastream/formats/parquet.md

```diff
@@ -39,46 +39,71 @@ To use the format you need to add the Flink Parquet dependency to your project:
 	<version>{{< version >}}</version>
 </dependency>
 ```
- 
+
+For reading Avro records, parquet-avro dependency is required additionally:
+
+```xml
+<dependency>
+    <groupId>org.apache.parquet</groupId>
+    <artifactId>parquet-avro</artifactId>
+    <version>${flink.format.parquet.version}</version>
+    <optional>true</optional>
+    <exclusions>
+        <exclusion>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+        </exclusion>
+        <exclusion>
+            <groupId>it.unimi.dsi</groupId>
+            <artifactId>fastutil</artifactId>
+        </exclusion>
+    </exclusions>
```

Comment:
Are these exclusions only needed when using this in combination with `flink-parquet` ? Or can we already exclude these by default?

Reply:
The exclusions are defined in the pom.xml as optional in the flink-parquet sub module. I think there were some known conflict.

Reply:
@zentol may I have your advice for this case? Thanks.

Reply:
Neither fastutil nor hadoop-client (or any hadoop dependency for that matter) is marked as `optional`, but `provided`.
fastutil is provided by flink-dist, while for hadoop we expect users to provided.

These exclusions don't really make sense to me.

Reply:
Ah now I see what you mean. flink-parquet contains an optional dependency on parquet-avro, and it has this set of exclusions.

Reply:
The fastutil exclusion in flink-parquet, or rather the `provided` dependency on it, doesn't make sense because fastutil isn't actually bundled by flink-dist.

Reply:
As for the hadoop-client exclusion in flink-parquet, it's a bit odd, and the only explanation I can offer is that it exists for dependency convergence.
An exclusion in the user application somewhat makes sense because we don't want it to be bundled, but imo it would be more appropriate to set it to provided somehow.

Reply:
Thanks for the suggestion. Beyond the reasons you mentioned above, what I understood additionally with the exclusions is that` flink-parquet` does not call any APIs and does not consume any functionality of the both excluded direct dependency `hadoop-client` and `fastutil`. Therefore, it is up to the user whether define them as provided or any other scopes in their own pom.xml. The Flink document just points out how to define the dependency of `parquet-avro`. In this case, with exclusions makes more sense.